Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            PersonalInfo();
+            PersonalInfo();
+        }
+
+        public static void PersonalInfo(){
+
+            string name = WholeName(message: "Input your name: ");
+            string surname = WholeName(message: "Input your surname: ");
+            int age = Age(message: "Input your age: ");
+            float height = HeightAndWeight(message: "Input your height in meters: ");
+            float weight = HeightAndWeight(message: "Input your weight in kg: ");
+            float bmi = CalculateBMI(weight: weight, height: height);
+
+            Console.WriteLine(name + " " + surname + " " + "is" + " " + age + " years old, his weight is " + weight + " kg and his height is " + height + " m and his/her BMI is: " + bmi);
+
+        }
+
+        public static string WholeName(string message)
```

Comment:
WholeName should be renamed to PromptString, because it doesn't matter what you get (a name, brand, nickname, etc). It's just a contactless string. It gets meaning only when is assigned to a variable. And in your case the variables are named right.

Reply:
Done!